.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: perl
+dist: trusty
```

Comment:
default dist is Xenial now, and Xenial only supports 5.22, 5.24, 5.26, 5.28 and 5.30.
https://docs.travis-ci.com/user/reference/xenial/#perl-support